nerfstudio/process_data/video_to_nerfstudio_dataset.py

```diff
@@ -41,6 +41,10 @@ class VideoToNerfstudioDataset(ColmapConverterToNerfstudioDataset):
     """Feature matching method to use. Vocab tree is recommended for a balance of speed
     and accuracy. Exhaustive is slower but more accurate. Sequential is faster but
     should only be used for videos."""
+    random_seed: Optional[int] = None
+    """Random seed to select video frames"""
```

Comment:
Is this random seed for select training set? maybe make it more clear in the comments